frontend/packages/integration-tests-cypress/views/common.ts

```diff
@@ -1,2 +1,5 @@
 export const resourceStatusShouldContain = (desiredStatus: string) =>
   cy.contains('[data-test="status-text"]', desiredStatus);
+
+export const waitForResourceStatusToContain = (desiredStatus: string) =>
+  cy.contains('[data-test="status-text"]', desiredStatus, { timeout: 30000 });
```

Comment:
The default timeout is 30 seconds so I don't think this is required.

Reply:
The default time for contain command (4 seconds - https://docs.cypress.io/api/commands/contains.html#Arguments), is not enough to get the desired status.

Reply:
https://github.com/openshift/console/blob/master/frontend/packages/integration-tests-cypress/cypress.json#L13
We have overridden that in the configuration